### PR TITLE
Fixed dotCMS/core#9730 by introducing fix-task that updates invalid asset-type in identifiers

### PR DESCRIPTION
**Pull request by agomez-dotcms**

### CODE REVIEW

File: src/com/dotmarketing/fixtask/tasks/FixTask00015FixAssetTypesInIdentifiers.java [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+								"SELECT * FROM "+ tableName +" WHERE identifier.id = identifier"+
+							")";
+
+						Logger.debug(MaintenanceUtil.class,"Task 15: Updating from identifier type " + asset + " : " + identifiersToUpdate);
+
+						dc.setSQL(identifiersToUpdate);
+
+						try {
+							dc.loadResult();
+						} catch (DotDataException e) {
```
Comment (dsilvam) [inline src/com/dotmarketing/fixtask/tasks/FixTask00015FixAssetTypesInIdentifiers.java]: What's the reason for this catch? and if we want to leave it there, shouldn't it be affecting the total that is set in FixAssetsProcessStatus.setErrorsFixed(total)?

File: src/com/dotmarketing/fixtask/tasks/FixTask00015FixAssetTypesInIdentifiers.java [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+					}
+				}
+
+				FixAssetsProcessStatus.setErrorsFixed(total);
+
+				FixAudit Audit = new FixAudit();
+				Audit.setTableName("identifier");
+				Audit.setDatetime(new Date());
+				Audit.setRecordsAltered(total);
+				Audit.setAction("task 20: Fixed DeleteOrphanedIdentifiers");
```
Comment (dsilvam) [inline src/com/dotmarketing/fixtask/tasks/FixTask00015FixAssetTypesInIdentifiers.java]: should be 'task 15' with its description here

**Review by dsilvam — CHANGES_REQUESTED** [review]

File: src/com/dotmarketing/fixtask/tasks/FixTask00015FixAssetTypesInIdentifiers.java (hunk 1 above)
Comment (agomez-dotcms) [inline src/com/dotmarketing/fixtask/tasks/FixTask00015FixAssetTypesInIdentifiers.java]: I think the try/catch may be removed. query should not fail unless there is a fatal error (such as network / database connection issue). I will remove it from FixTask00020DeleteOrphanedIdentifiers as well.

File: src/com/dotmarketing/fixtask/tasks/FixTask00015FixAssetTypesInIdentifiers.java (hunk 2 above)
Comment (agomez-dotcms) [inline src/com/dotmarketing/fixtask/tasks/FixTask00015FixAssetTypesInIdentifiers.java]: Absolutely... thanks for the comments!

**Review by dsilvam — APPROVED** [review]

**Review by jgambarios — APPROVED** [review]

**Pull request merged**